6.4_remember_remember.py

```diff
@@ -0,0 +1,29 @@
+"""
+Writer: Ranel Ben Simman Tov
+"""
+import numpy as np
+from PIL import Image
+
+IMAGE_PATH = "code.png"
```

Comment:
Usually code and resources like images are not saved in the same directory.
It is costume to have images library and use relative path in your code.
No need to change it, but keep in mind for future times.